components/camel-google/camel-google-pubsub-lite/.gitignore

```diff
@@ -0,0 +1,7 @@
+.idea
```

Comment:
This file could be removed.

Reply:
Fixed in [8bfe10f](https://github.com/apache/camel/pull/13630/commits/8bfe10f92a9fd68d9d9fa417c5e906aca3b20d7a)